sssd_test_framework/roles/ldap.py

```diff
@@ -1759,3 +1776,109 @@ def __get_info(self, info: str | NFSExport | LDAPAutomountMap | DeleteAttribute
             return info.name
 
         return info
+
+
+class LDAPPasswordPolicy(object):
```

Comment:
Shouldn't this be a subclass of `GenericPasswordPolicy`?

Reply:
In generic.py , the class `GenericProvider(ABC, MultihostRole[BaseHost]):` contains the same methods as all the other providers, it just passes the call to the specific provider and it should be there. =D